src/sentry/features/temporary.py

```diff
@@ -351,6 +351,8 @@ def register_temporary_features(manager: FeatureManager):
     # Limit project events endpoint to only query back a certain number of days
     manager.add("organizations:project-event-date-limit", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
     manager.add("organizations:project-templates", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=False)
+    # Enable the new quick start guide
+    manager.add("organizations:quick-start-updates", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=False)
```

Comment:
Is this purposely FeatureHandlerStrategy.INTERNAL and not FLAGPOLE?

Reply:
copy and paste 😄 great feedback..thanks